versioned/version0_0_10.ts

```diff
@@ -30,93 +31,47 @@ class Version0_0_10 extends Versioned {
-                  execute_lambda: {
-                    metadata: meta,
-                    lambda: michelsonCode,
-                  },
-                };
-              }
-              case "contract": {
-                const p = new Parser();
-                const michelsonCode = p.parseMichelineExpression(
-                  x.values.lambda
-                );
-                let meta = !!x.values.metadata
-                  ? convert(x.values.metadata)
-                  : null;
-                return {
-                  execute_lambda: {
-                    metadata: meta,
-                    lambda: michelsonCode,
-                  },
-                };
-              }
-              case "fa2": {
-                const parser = new Parser();
+    if (batchOp === undefined) batchOp = t.wallet.batch();
 
-                const michelsonCode = parser.parseMichelineExpression(
-                  generateFA2Michelson(
-                    this.version,
-                    x.values.map(value => ({
-                      walletAddress: cc.address,
-                      targetAddress: value.targetAddress,
-                      tokenId: Number(value.tokenId),
-                      amount: Number(value.amount),
-                      fa2Address: value.fa2Address,
-                    }))
-                  )
-                );
+    const content = proposals.transfers
+      .map(x => this.mapTransfer(x, cc))
+      .filter(v => Object.keys(v).length !== 0);
+
+    if (content.length > 0) {
+      const params = cc.methodsObject.create_proposal(content);
+      batchOp.withContractCall(params);
+
+      if (isSigning) {
+        const storage = toStorage(
+          this.version,
+          await cc.storage(),
+          BigNumber(0)
+        );
+        const proposalId = storage.proposal_counter.plus(proposalIdOffset);
+        return await this.signProposal(
+          cc,
+          t,
+          proposalId,
+          true,
+          isResolving,
+          batchOp,
+          content
+        );
+      }
+    }
+
+    let params = cc.methods.create_proposal(content).toTransferParams();
 
-                return {
-                  execute_lambda: {
-                    metadata: convert(
-                      JSON.stringify({
-                        contract_addr: x.values[0].targetAddress,
-                        payload: x.values.map(value => ({
-                          token_id: Number(value.tokenId),
-                          fa2_address: value.fa2Address,
-                          amount: Number(value.amount),
-                        })),
-                      })
-                    ),
-                    lambda: michelsonCode,
-                  },
-                };
-              }
-              default:
-                return {};
-            }
-          })
-          .filter(v => Object.keys(v).length !== 0)
-      )
-      .toTransferParams();
     let op = await t.wallet.transfer(params).send();
```

Comment:
sent batchop instead
```suggestion
    const op = await batchOp.send();
```